README.md

```diff
@@ -1,14 +1,63 @@
 # Chuck Norris Joke API
 
+## Overview
 A simple project created to practice working in a team environment using Agile methodologies.
 
-(Note - include the icons for the stack + team members)
+**Live link:** [https://chingu-voyages.github.io/v42-toucans-team-02/](https://chingu-voyages.github.io/v42-toucans-team-02/)
 
-Here is Gillian's Readme comment for testing:
-Hello team!
 
-This is Lyle's Readme comment for testing:
-Hey guys... I mean gals...
+## Features
+* Clicking nav bar items drop user to appropriate page elements
+* Searching for a joke consumes the Chuck Norris API and returns an object
+* Color schemas abide by contrast ratios for accessibility requirements
+* etc...
 
-Last, but not least.........Melissa!
-Here's my Readme comment for testing :)
\ No newline at end of file
+
+## Running The Project
+Live verson:
+[Chuck Norris Jokes](https://chingu-voyages.github.io/v42-toucans-team-02/)
+
+From the repo:
+
+  1. Clone this project locally
+  2. ...
+  3. ...
+  4. Invite friends or family over and tell some jokes!
+
+
+## Tech Used
+<p align="left">
+    <img
+    src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg"
+    alt="html5"
+    width="40"
+    height="40"
+    />
+    <img
+    src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg"
+    alt="css3"
+    width="40"
+    height="40"
+    />
+    <img
+    src="https://raw.githubusercontent.com/devicons/devicon/master/icons/javascript/javascript-original.svg"
+    alt="javascript"
+    width="40"
+    height="40"
+    />
+    <img
+    src="https://www.vectorlogo.zone/logos/git-scm/git-scm-icon.svg"
+    alt="git"
+    width="40"
+    height="40"
+    />
+</p>
+
+
+
+## Contributors
+The Team |  GitHub  | LinkedIn
+:---:  |  :---:  |  :---:
+Lyle Trethewey  |  [![git hub logo](https://github.com/chingu-voyages/v42-toucans-team-02/images/github.png "Lyle's github")](https://github.com/LyleT)  |   [![linkedin logo](https://github.com/chingu-voyages/v42-toucans-team-02/images/linkedin.png "Lyle's LinkedIn")](https://www.linkedin.com/in/lyle-trethewey/)
```

Comment:
Hi Melissa - these images are not rendering. Are you able to fix this problem? You can view the readme by clicking the three dots under the review changes button. 8)

Reply:
@LyleT - will they only display once the merge is complete? The URLs are pointing to where we are housing the images inside the repo.